api/server/server.go

```diff
@@ -855,7 +856,18 @@ func (s *Server) startGears(ctx context.Context, cancel context.CancelFunc) {
 
 	server := s.svcConfigs[WebServer]
 	if server.Handler == nil {
-		server.Handler = &ochttp.Handler{Handler: s.Router}
```

Comment:
Prevents traces of / path from showing up on the exporter/metrics. 
Need to scrap AlwaysSample by default and move to per span sampling eventually.